<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SC-Walidator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="tools-check-spelling">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="separator"/>
    <addaction name="actionLoadall"/>
    <addaction name="actionSaveall"/>
    <addaction name="separator"/>
    <addaction name="actionRoot"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Tasks</string>
    </property>
    <addaction name="actionAutosave"/>
    <addaction name="actionAutorestore"/>
    <addaction name="separator"/>
    <addaction name="actionClearall"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="system-run">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Run all tasks</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRoot">
   <property name="icon">
    <iconset theme="format-indent-more">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select root directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAutosave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="appointment-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save tasks on exit</string>
   </property>
  </action>
  <action name="actionAutorestore">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-open-recent">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Restore tasks on startup</string>
   </property>
  </action>
  <action name="actionClearall">
   <property name="icon">
    <iconset theme="edit-clear">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Clear all tasks</string>
   </property>
  </action>
  <action name="actionSaveall">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save all tasks</string>
   </property>
  </action>
  <action name="actionLoadall">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Load saved tasks</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
